ydb/library/yql/dq/actors/protos/dq_stats.proto

```diff
@@ -312,12 +316,21 @@ message TDqStageStats {
     TDqStatsAggr EgressBytes = 30;
     TDqStatsAggr EgressRows = 31;
 
-    TDqStatsMinMax FirstRowTimeMs = 13;
-    TDqStatsMinMax FinishTimeMs = 14;
-    TDqStatsMinMax StartTimeMs = 21;
-    uint64 DurationUs = 15; // microseconds from min(task_first_row_time) to max(task_finish_time)
+    reserved 13; // FirstRowTimeMs
```

Comment:
Are we confident that this stat don't need here anymore?

Reply:
Long story here:

On Task level FirstRowTimeMs was never used actually. So, I've dropped this misleading field and left Start and Finish only, which seems to be self-descriptive. Task tries to report first input row timings as StartTime. "Old" StartTime (time when task was started) is reported if traffic timings are not available. It is not useful - all tasks start at about the same time, still is reported as some kind of approximation

On Stage level (aggregated stats) FirstRowTimeMs is dropped as well. Other metrics - Start and Finish are changed from Min/Max only to full set (+ Count and Sum). Sum has no meaning for timings, but Count has and Average = Sum/Count is also useful.